sdk/clientcore/core/src/test/java/io/clientcore/core/http/pipeline/SetUserAgentPolicyTests.java

```diff
@@ -99,11 +99,11 @@ private static Stream<Arguments> userAgentAndExpectedSupplier() {
             Arguments.of(new UserAgentPolicy("AutoRest-Java"), "AutoRest-Java"),
 
             // Tests using SDK name and version with platform information and without application ID
-            Arguments.of(new UserAgentPolicy(null, sdkName, sdkVersion),
+            Arguments.of(new UserAgentPolicy(new UserAgentOptions(sdkName, sdkVersion)),
```

Comment:
nit: unrelated to the PR but happened to notice it here. This test class should be named `UserAgentPolicyTests` to match the policy name.